src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -274,7 +274,7 @@ def load_arguments(self, _):
         c.argument('backend_pool_name', help='Name to use for the backend pool when creating a new load balancer or application gateway.')
         c.argument('backend_port', help='When creating a new load balancer, backend port to open with NAT rules (Defaults to 22 on Linux and 3389 on Windows). When creating an application gateway, the backend port to use for the backend HTTP settings.', type=int)
         c.argument('load_balancer', help='Name to use when creating a new load balancer (default) or referencing an existing one. Can also reference an existing load balancer by ID or specify "" for none.', options_list=['--load-balancer', '--lb'])
-        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', help='SKU when creating a new Load Balancer.', options_list=['--lb-sku'], arg_type=get_enum_type(LoadBalancerSkuName, default='Basic'))
+        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', help='SKU when creating a new Load Balancer.', options_list=['--lb-sku'], arg_type=get_enum_type(LoadBalancerSkuName))
```

Comment:
Is there no longer a default or is it dynamically determined?  If it is dynamic we should explain that in the help.